### PR TITLE
Aad 729 reminder notifications

### PR DESCRIPTION
**Pull request by CathyGroom**
add new endpoint for getting task reminders by apprentice

### CODE REVIEW

File: src/SFA.DAS.ApprenticeProgress.Application/Tasks/Queries/GetTaskRemindersByApprenticeshipId/GetTaskRemindersByApprenticeshipIdHandler.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                    reminder => reminder.TaskId,
+                    (task, reminder) => new { task, reminder })
+                .Where(r => r.reminder.Status == Domain.Entities.ReminderStatus.NotSent)
+                .AsNoTracking()
+                .AsSingleQuery()
+                .ToListAsync(cancellationToken);
+
+            foreach(var task in tasks)
+            {
+
```
Comment (alireid) [inline src/SFA.DAS.ApprenticeProgress.Application/Tasks/Queries/GetTaskRemindersByApprenticeshipId/GetTaskRemindersByApprenticeshipIdHandler.cs:38]: Tidy

Comment (CathyGroom) [reply src/SFA.DAS.ApprenticeProgress.Application/Tasks/Queries/GetTaskRemindersByApprenticeshipId/GetTaskRemindersByApprenticeshipIdHandler.cs:38]: I also modified the Where clauses slightly to only include ToDo tasks and to only exclude Dismissed reminders.

**Review by alireid — APPROVED** [review]

**Review by alireid — APPROVED** [review]

**Review by alireid — APPROVED** [review]
Visually looks great

**Review by alireid — CHANGES_REQUESTED** [review]
Coverage...

**Review by alireid — APPROVED** [review]

**Pull request merged**